src/main/java/com/quizte/entity/User.java

```diff
@@ -0,0 +1,63 @@
+package com.quizte.entity;
+
+import com.quizte.dictionary.Role;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.CollectionTable;
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.Email;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "Users")
+@Getter
+@Setter
+@ToString
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    Long id;
+
+    @Email
+    @Column
+    String email;
+
+    @Column
+    String password;
+
+    @Column
+    String firstName;
+
+    @Column
+    String lastName;
+
+    @Column
+    String photoUrl;
+
+    @Column(name = "is_active")
+    private Boolean active;
+
+    @CollectionTable(name = "user_role", joinColumns = {@JoinColumn(name = "user_id")})
+    @ElementCollection(fetch = FetchType.EAGER)
+    @Enumerated(EnumType.STRING)
+    Set<Role> roles = new HashSet<>();
```

Comment:
let's keep it simple: user can have only one role. than we can make it a simple column and avoid extra join for roles